test/cli/symbol-table-json-alias/symbol-table-json-alias.sh

```diff
@@ -1 +1 @@
-main/sorbet test/cli/symbol-table-json-alias/symbol-table-json-alias.rb -p symbol-table-json | grep aliasTo -B 5 | head -n 5
+main/sorbet test/cli/symbol-table-json-alias/symbol-table-json-alias.rb -p symbol-table-full-json | grep aliasTo -B 5 | head -n 5
```

Comment:
I am not exactly sure what this test is exercising so couldn't be sure if leaving this as `symbol-table-json` would be ok. It feels like it should be, please advise if so.